templates/rke2-airgap/overlay/opt/corral/rke2/download-rke2.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+apt install -y jq
+
+kubernetes_version=${CORRAL_kubernetes_version/+/"%2B"}
+mkdir /root/rke2-artifacts && cd /root/rke2-artifacts/
```

Comment:
`/root/rke2-artifacts` should probably be configurable

Reply:
How exactly to you mean? Right now the packages downloaded are dependent on what the Kubernetes version is set to by the package.

Reply:
Probably something simple like:
```suggestion
ARTIFACT_ROOT=/root/rke2-artifacts
mkdir -p $ARTIFACT_ROOT && cd $ARTIFACT_ROOT
```